src/main/java/com/alexx/employeesAndDepartments/controllers/DepartmentController.java

```diff
@@ -37,28 +38,25 @@ public ResponseEntity<DepartmentDTO> editDepartment(@RequestBody DepartmentDTO d
     @DeleteMapping("/departments/{departmentId}")
     @Operation(summary = "Delete department", description = "Delete an existing department, ID of department in path required")
     public ResponseEntity<Void> deleteDepartment(@PathVariable("departmentId")
-                                                     @Min(value = 1, message = "ID of department must be positive.")
-                                                     @Parameter(description = "ID of the existing department to remove")
-                                                     Integer id) {
+                                                 @Parameter(description = "ID of the existing department to remove")
+                                                 UUID id) {
         departmentService.deleteDepartment(id);
         return ResponseEntity.status(HttpStatus.NO_CONTENT).build();
     }
 
     @GetMapping("/departments/{departmentId}")
     @Operation(summary = "Get department", description = "Get an existing department with its list of employees, ID of department in path required")
     public ResponseEntity<DepartmentDTO> getDepartment(@PathVariable("departmentId")
-                                                           @Min(value = 1, message = "ID of department must be positive.")
-                                                           @Parameter(description = "ID of the existing department to get")
-                                                           Integer id) {
+                                                       @Parameter(description = "ID of the existing department to get")
+                                                       UUID id) {
```

Comment:
Please use String as a parameter in requests and then internally convert it to UUID (UUID.fromString(...)

Reply:
got it, will fix